gulpfile.js

```diff
@@ -30,7 +30,7 @@ gulp.task('styles', function () {
         .pipe(gulp.dest('./css'));
 });
 
-gulp.task('watch', ['scripts', 'styles'], function() {
-    gulp.watch('js/*.js', ['scripts']);
-    gulp.watch('scss/*.scss', ['styles']);
-});
+gulp.task('watch', function() {
+    gulp.watch('js/*.js', gulp.series('scripts'));

```

Comment:
There seems to be an issue with the update where once you make a change to `scripts.js` it ends in an infinite loop as it also tracks the change in `scripts.min.js` where it is being compiled to.

Reply:
Whoops, missed that.
Folder structure can be definitely improved but still, as for now, the gulpfile does not seems to handle automatically "custom JS/SCSS files" (other than the default scripts.js and styles.scss).

The easiest solution is then to watch only scripts.js and styles.scss.

Reply:
Thanks for the quick fix! And I completely agree, there is room for a better folder structure (src / dist or something along those lines perhaps), however I opted for simple as this is only a single page portfolio and would allow it to be more easily edited by those who pick it up.